ocs_ci/ocs/resources/bucket_policy.py

```diff
@@ -152,19 +162,27 @@ def gen_bucket_policy(
     )
     ver = datetime.date.today().strftime("%Y-%m-%d")
 
+    principal = principal_property if principal_property else "Principal"
+    effect = effect if effect else "Allow"
+    action = action_property if action_property else "Action"
+    resource = resource_property if resource_property else "Resource"
+
     logger.info(f"version: {ver}")
+    logger.info(f"Principal: {principal}")
     logger.info(f"principal_list: {principals}")
+    logger.info(f"Action: {action}")
     logger.info(f"actions_list: {actions_list}")
+    logger.info(f"Resource: {resource}")
     logger.info(f"resource: {resources_list}")
```

Comment:
Please combine these line to avoid confusion.

Suggested changes:
```
    logger.info(f"{principal}: {principals}")
    logger.info(f"{actions}: {actions_list}")
    logger.info(f"{resource}: {resources_list}")
```

Reply:
Updated these lines.